src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -6,12 +6,19 @@
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Methodologies;
 using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
 using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.ManageContent;
-using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Methodology;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
 using GovUk.Education.ExploreEducationStatistics.Publisher.Model;
+using ContactViewModel = GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.ContactViewModel;
```

Comment:
Out of interest, where have all these `usings` popped up from?

Reply:
I think because there are duplicate view models in `Content.Services.ViewModels`, when I added `ExternalMethodologyViewModel` it having to explicitly declare all of these.

Reply:
This has highlighted an issue - we shouldn't be depending on Content.Services in Admin, so I've made a new `ExternalMethodologyViewModel` that lives in `Admin.ViewModels`.

Reply:
I've also made a clone of `PublicationSummaryViewModel` into Admin so we can remove the dependency on `Content.Services.ViewModel` from `PublicationController` and `PublicationService` in Admin.